db/seeds.rb

```diff
@@ -40,15 +40,15 @@
 
 minimal_mailing = Address.find_or_initialize_by(
   snap_application: minimal_application,
-  street_address: "",
 )
 
 minimal_mailing.update!(
-  city: "",
-  zip: "",
+  street_address: "123 Main St.",
+  city: "Flint",
+  zip: "12345",
   county: "Genesee",
   state: "MI",
-  mailing: false,
+  mailing: true,
```

Comment:
updated this to make it more in line with reality: we get a mailing address for all people, but a residential address for some only. 

We validate all fields are present on a mailing address